tools/manage_clvm.py

```diff
@@ -282,6 +321,13 @@ def check(use_cache: bool) -> int:
         if file_fail:
             overall_fail = True
 
+    if missing_files != []:
+        overall_fail = True
+        print()
+        print("Missing files (run tools/manage_clvm.py build to build them):")
```

Comment:
Or I guess we could at the top do something like calculate `__file__` relative to `root` to make this automatic.  This also raises the point of renaming this file.

```suggestion
        print("Missing files (run tools/manage_clsp.py build to build them):")
```

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/14996#discussion_r1163237917